tests/e2e/samples/config/gcp-key.json.enc

```diff
@@ -0,0 +1 @@
kGFklVRlGOKG75IFQGsenZqpm1bJdcJOl514KkwdyW6wd9vhkwLqm1V2OgKgZfoBnncQSzrGhSd8NZXzgcdRUrqEw20G7jlw1pjDQW5uQElF1NzPNpZGjANf_gqWeV_TWSxfBQ1YPRuxpR-TOzYSGUNr4uNpefVIKbVXFEC0O-_gFDDV7EYeAFDHp6PxwbLeypOajny8SZ-5C_tFkMESJWX99XsukD-5jJGbj_8VuJ7_OI2BgrHM7bOekXsXpeq1_sMdKF68jh-lTAF-rUIiUoFR2dU8i5j9WQLrPA9544P04X3bxF-GbU6YpCOFSc-rjudm4F7eiZevhBL-BqfuBganUKof2IFKw1H8iYJRNe2FJ6DVYBiCOGwcDizPqxCaorto01B4WvcDA96ebuRLGIgd3t_Vqp1lPrTXYxxJtW7nFfnoCVqe5MRD5B4xUdyvUHFUyD3EOlvQAxPIGdOA91TQCTyVDKxN_4LGcpb9H06EujHWUbwyUNzlX1clQCnS9rIE1-_tyXqvfOHzPQj5jsPwYqXumWBkFyz9rGAfR-Rx5jk9-7kQyK6ykT4MaSSVTUfgJ7FtpQQM2czo15wDMtAKP3GuPTwtff6c7Xsy9cFBhwkO7zi7B6X9_6mo8MM_mggk1GX6FHjkDtBfuCj38ByIWUHoDWMouKgCQ_Ms1wMybe_TzV0vxyZRiR4050rTFbeZVAyaMV_vV4xoePoqURW2wSIv6p0QF9WAwVdd7oVnxnQCvTtErPQeHALLMnTQ3pe4QxVoSHLEGcCiBkwmOSAIqOBD9P-kC_DAj8SHVm8hBKATVJrEunZgW2JAGH74b5_lsOpF-UrYJGFD3zez2uKz5ru9r7FD8iaa7VIV0sDag59PK3G-a4zn-oHu8JeB5TU4tJ0Cf8KxiDDCavUMLsC25S52MW1DbiBgmXnFNXLUTDjBao7952nrxjJ7LKgjjfyHv21_yhQwkKBEbDbUXCI44w_-ke1pfkJvogXFTbKaOPZrrgbOwgRLLTLMFJ_VnWMAm-xA0yT7jWW8TM_URO1ul69Wg3DnEh5D2zciQP2xPRQ90UufY302fOR7G5otHPbMH4GUYAkkEiXav2dF7PFPthjvC8y2Hrv2KzqB-ZZYV7IC5Z9r1dpg_JW9p8jIPave55vn0nHz3vbivMVfxMGo9CvgbbhyJJeIjAP3-f3oVwU4HHH21HLs7HD4reoIknbGoCqy3Z9sSfgAu7beckS65eZy0tNKWfLM8L_0Jt6k2-uol_9PWw_4aEHUQC-9EQLf8_SESTz_GRftHTxDu8V0ei5diA71TURPREF12Mof0Iixyvcdi0w9T96HWITyvjti-LtMDqosz_rzrvK07n_a_nqukTunLTmUizQP07BAr9siOL372ctzDuzD0PGJmJigo379vDBmo6_ZlXbEBVIyQ7dRGdSOIiYJPJmXEzeRgdEMuF3-R9_PX3JAzSNxClC0Udf2JG60tSQj9lJ8_i5QXA-47rRSjt2pTy-8aepfAbtoM1xQ7ZzgD-f3x9LWesi3x4a6l4CUyf6v-JaNopgP3JjrdocXGNerCQVn0B2ZhGpRWXeN1euuyRPYxsnnRI6VlG6DjpOUW-yE4noqeCDK1TfhWHyRVDorr20dgzJwY6YwrBh6nRO9oD3j4v8pNCWtIYZsl3O1bhCykLt3r01Ap8wS-MfO2Kc1F4NFHFpQmZZ2180HdQXLWTkqMoeVHW1IAJaWf1M2y7Sm-sT9guGXhu5CedkhUBXOGxdZ2FlboS03KtjcuPgXBd0vBwOxT2QfAJO7eO6uCU5O7dqzPPwIISZ7CERddiJ4M_qpnluF9nYg9RO1FTfRWfHrFfWRZo0ywAnhlWtyp1n66cz_vMh1NM-uCqmzEETWJmvy4PmZ-btEqOZQnU-4c82Hw4AviJGv9ZOZaGfpRiqtkLdE3bvrr0bvksJ0-lF7bNg05C9K4duLnizl5QAiG5mQWDmyaTK5qhGsutKMEja1Giz3kngxnKQdFiRlynKoXr0ONa-jSO0eaUFDJPhI-wDgG7xOE7gG0UJdrn-becxnsjDLTGdSpvkQZludKHMYJXdiUjtagRzkESUMJsiHRWL3BhfRGduwMJbPALDoiaonisV48qoJsoW5jkpj__b3N3ymwS3x2c6dZY317efqrFKlXOpGopmTJ2GEa0UgNFV8Np-Onxdm_P5pNRTi-mFdJoGcCITepHR4jxzmvAJ6g1Q8zqgH4rMyhRr-d7nSOz6EEYkzXU62TBUFix9PNFJeu9RRln1rKHX0lSmbHu-KgsVEBgaaXKMZiaez37zgPwc_x6xtW2bJKAjYpLSXPIMRai8NRmzvorT26K88zkzc-vVtmLpAlBNiIyoag-N_F8kjJtD3XI8mJ4gDE9wOUiv8eUwaK_mf60mjCfhhXYAXGY2cWxHmW-PWWAJ7DQ-g1o0WKAZmU91Zp2LFcVXCxe2jABtpTPo04-EUhvJzm2LOpbgjxkdajgKtptGAkNoC_zqGEKCHprUxzVlPdqu9ZFv0a9HI1vd0QlcmqkWjS95mmsNGJoVVVFfB9Ux1g81hor2K3y8HstUYfANwf_vv2_IbbeEHi81xNGzM_qE8cIzdJ0C5s27jT7V9FGAQ47ObyXXLZnVZEu9oU1qEBcORXxmSSkaqbeoJT4flv1y0ZSvVYXDP_fv7DeewQpkwoNONl1RnuYfD3j6gdwVSqD_0Cgg_T06Ay5Fv8K6YUmDcmLwy024X5J-fm1AIHGXM1konkzuw1UFCv4q0WrXCiiEZR7tO48YNpJDkv6uV9efE15YVlw8wg3lxN6kbYFH9WTdRDrc82y5rOakpDbgqfpcRXFqifEh0qws6DCgOYjZH4QTtEq7HZxOYM4xbbl4bXm8n4r41rvYk6XIng4l6LyoKCR5qfuyL8JWvjO2epNr6AW3j9_4Ul11rzTfxkEl896FyuNAATbGmOcBGZ7kqxIPKWLT7uTrdd6qakgECLjdcPuI=
```

Comment:
configuration file encrypted with the key stored in env var `COOKIECUTTER_GCP_CONFIG_ENCRYPTION_KEY`. Please note that in order to run tests locally, you need to set up this variable.